samples/bot-sso-adaptivecard/nodejs/README.md

```diff
@@ -0,0 +1,174 @@
+﻿---
+page_type: sample
+description: Microsoft Teams Bot-SSO-Adaptivecard
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "21/01/2023 12:54:21 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-sso-adaptivecard-nodejs
+
+---
+
+# SSO for your Adaptive Cards Universal Actions
+
+This sample code demonstrates how to get enable SSO authentication for your Adaptive Cards Universal Actions in bot.
```

Comment:
This sample code demonstrates how to get enable SSO authentication for your Adaptive Cards.

Reply:
Refer this Readme:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/653/files#diff-f809f3228a48a2949d16bed7cc704a768823d2bc6cfa3ccaf27af1b4ac6de34f